composer/algorithms/seq_length_warmup/seq_length_warmup.py

```diff
@@ -306,16 +310,13 @@ def _activate_model(self, state: State, logger: Logger) -> None:
 
             # This error/state.grad_accum handling mimics the logic in trainer._train_batch().
             except RuntimeError as e:
-                if 'CUDA out of memory' in str(e):
+                if state.adaptive_gradient_accumulation and _is_cuda_oom(e):
+                    log.debug((f"Rank {dist.get_global_rank()} OOM'd."))
                     found_cuda_oom = 1
                 else:
                     raise
```

Comment:
cc: @alextrott16 on whether they want to require `grad_accum=auto` for seq length warmup oom protection.

If so, then maybe we should print something here about setting ga=auto for SLW once it OOMs.

Reply:
Note that SLW works without `grad_accum=auto` (it existed before aga did) and I don't think we should require it